docs/sources/shared/configuration.md

```diff
@@ -4755,6 +4760,10 @@ Configures the `server` of the launched module(s).
 # CLI flag: -server.grpc-conn-limit
 [grpc_listen_conn_limit: <int> | default = 0]
 
+# Enables PROXY protocol.
```

Comment:
Do we need to rebase this PR for these other `configuration.md` changes?

Reply:
these docs changes appear because we updated the version of dskit. and as long as dskit update is a part of this ticket, I believe it's ok if we have docs changes here as well